databricks/resource_databricks_group_role.go

```diff
@@ -0,0 +1,120 @@
+package databricks
+
+import (
+	"fmt"
+	"github.com/databrickslabs/databricks-terraform/client/model"
+	"github.com/databrickslabs/databricks-terraform/client/service"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+	"strings"
+)
+
+func resourceGroupRole() *schema.Resource {
```

Comment:
this resource might have an alias - `databricks_group_instance_profile` :) because here we semantically mix two things and confuse outside user

Reply:
Great point, I always found this quite confusing as well... I will switch it to databricks_group_instance_profile